code/game/objects/items/weapons/autolathe_disks.dm

```diff
@@ -821,7 +825,7 @@
 	)
 
 /obj/item/weapon/computer_hardware/hard_drive/portable/design/guns/dallas
-	disk_name = "PAR - .25 Dallas"
```

Comment:
Leave the default name. Also, starting ANYTHING with number, dot or something like this is bad for readability.

Reply:
Resolved.